Sources/SmithyIdentityAPI/AWSCredentialIdentityResolver.swift

```diff
@@ -1,9 +0,0 @@
-//
```

Comment:
This is moved to `Sources/SmithyIdentity/AWSCredentialIdentityResolver.swift`, but git picked it up as deletion because the new file has new changes.